### PR TITLE
BRITECHSRM-397: Speaker rating update with patch

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: app/services/ApiService.js [hunk 1]
```diff
@@ -37,21 +37,31 @@ class Api {
         });
     }
 
-    get(uri) {
+    getAuthedHeaders() {
```
Comment (pallenSL) [inline app/services/ApiService.js]: "Authed" is not a word!

File: app/services/ApiService.js [hunk 2]
```diff
@@ -37,21 +37,31 @@ class Api {
...
         const headers = new Headers();
         headers.append('Authorization', `Bearer ${AuthService.getAccessToken()}`);
-        const requestData = { headers };
+        return headers;
+    }
 
+    getAuthedJsonHeaders() {
```
Comment (pallenSL) [inline app/services/ApiService.js]: Don't make up new words for us all to learn!

**Comment by pallenSL** [comment]
Other than the new words, :shipit:

**Comment by jebbinSCL** [comment]
Renamed

**Comment by pallenSL** [comment]
:shipit:

File: app/services/ApiService.js [hunk 3]
```diff
@@ -37,21 +37,33 @@ class Api {
...
+        headers.append('Content-Type', 'application/json');
+        return headers;
+    }
+
+    get(uri) {
+        const requestData = { headers: this.getAuthorizationHeaders() };
         return this.performRequest(uri, requestData);
     }
 
     put(uri, data) {
```
Comment (csscottlogic) [inline app/services/ApiService.js]: Is PUT now ever used from the front end?

**Pull request closed without merging**